Symfony/src/AppBundle/Controller/DashboardController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DashboardController extends Controller
+{
+    /**
+     * @Route("/dashboard", name="dashboard")
+     */
+    public function showAction()
+    {
+        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
+        $usr = $this->get('security.token_storage')->getToken()->getUser();
+        $firmwares = $this->getDoctrine()
+            ->getRepository('AppBundle:FirmwareConfig')
+            ->findByOwner($usr->getId());
+        $tier = $this->getDoctrine()
+            ->getRepository('AppBundle:Tier')
+            ->find($usr->getTier());
+
+        return $this->render('AppBundle:Dashboard:index.html.twig',
```

Comment:
If you need to split a line because the arguments don't fit, then you have to keep *each* argument on a separate line according to PSR rules. So:
```php
return $this->render('AppBundle:Dashboard:index.html.twig', 
     array('user' => $usr, 'firmwares' => $firmwares, 'tier' => $tier));
```
should become
```php
return $this->render(
    'AppBundle:Dashboard:index.html.twig', 
    array('user' => $usr, 'firmwares' => $firmwares, 'tier' => $tier)
);
```